api/specs.proto

```diff
@@ -165,6 +165,16 @@ message ContainerSpec {
 	// StopGracePeriod the grace period for stopping the container before
 	// forcefully killing the container.
 	Duration stop_grace_period = 9;
+
```

Comment:
I add the comments on the struct definition. it is just few lines below, is it OK?